src/monitor.rs

```diff
@@ -14,24 +14,24 @@ use crate::{
 ///
 /// Can be acquired with [`MonitorHandle::video_modes`].
 #[derive(Clone, PartialEq, Eq, Hash)]
-pub struct VideoMode {
-    pub(crate) video_mode: platform_impl::VideoMode,
+pub struct VideoModeHandle {
+    pub(crate) video_mode: platform_impl::VideoModeHandle,
 }
 
```

Comment:
Maybe add:
```rust
#[deprecated = "Renamed to `VideoModeHandle`"]
type VideoMode = VideoModeHandle;
```

And keep it around for a release cycle?

Reply:
I don't mind.

Reply:
Done.